app/Models/Feed.php

```diff
@@ -219,8 +228,8 @@ public function _keepHistory($value) {
 	public function _ttl($value) {
 		$value = intval($value);
 		$value = min($value, 100000000);
-		$value = max($value, -2);
-		$this->ttl = $value;
+		$this->ttl = abs($value);
```

Comment:
I am not sure to understand what the logic is when selecting ∞ / -1 for the TTL. Could you please explain?

Reply:
There is none. I've just forgot to remove that option from the drop-down :)